x-pack/plugins/apm/server/routes/alerts/rule_types/transaction_duration/get_transaction_duration_chart_preview.ts

```diff
@@ -89,7 +89,9 @@ export async function getTransactionDurationChartPreview({
         ...termFilterQuery,
         ...getParsedFilterQuery(searchConfiguration?.query?.query as string),
         ...rangeQuery(start, end),
-        ...getDocumentTypeFilterForTransactions(searchAggregatedTransactions),
+        ...getBackwardCompatibleDocumentTypeFilter(
+          searchAggregatedTransactions
+        ),
```

Comment:
I'm not sure the rename to `getBackwardCompatibleDocumentTypeFilter` makes sense until we have updated all APIs to use the response from the time_range_metadata API. That being said, I hope we do that soon, so I'm not strongly attached to either name

```suggestion
        ...getBackwardCompatibleDocumentTypeFilter(
          searchAggregatedTransactions
        ),
```

Reply:
btw. I think this function should eventually indicate that it's specifically a filter for 1 minute transaction metrics.

Reply:
> makes sense until we have updated all APIs to use the response from the time_range_metadata API.

Thats what i will continue to work on immediately after this. Due to FF being close, didn't wanted to increase the scope of a single PR.

> btw. I think this function's name should eventually indicate that it's specifically a filter for 1 minute transaction metrics.

As of now `backward-compatibility` in APM has only 1 meaning and that is 1 minute transaction metrics. I think this makes its obvious no ?